flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/SqlClientTest.java

```diff
@@ -205,6 +205,17 @@ public void testExecuteSqlFile() throws Exception {
         }
     }
 
+    @Test
+    public void testExecuteSqlWithHDFSFile() throws Exception {
+        List<String> statements = Collections.singletonList("HELP;\n");
+        String sqlFilePath = createSqlFile(statements, "test-sql.sql");
+        String[] args = new String[] {"-f", "hdfs:/" + sqlFilePath};
```

Comment:
Just use a invalid file path is enough. We don't need to create a new file here.

Should be `"hdfs://path/to/file"`?

Reply:
yes, u are rigth just need a path